acme/resource_acme_certificate_test.go

```diff
@@ -677,6 +710,30 @@ func testAccCheckACMECertificateProxyTestServer() (func(), error) {
 	}, nil
 }
 
+func testAccCheckACMECertificateS3ProxyTestServer(s3Bucket string, awsRegion string) (func(), error) {
+	target := fmt.Sprintf("%s.s3.%s.amazonaws.com",s3Bucket, awsRegion)
+
+	proxy := &httputil.ReverseProxy{
+        Director: func(req *http.Request) {
+            req.URL.Scheme = "https"
+            req.URL.Host = target
+            req.Host = target
+        },
+    }
```

Comment:
@kingcdavid looks like this definitely needs to be run through `gofmt`, as this brace is out of place as well.